Add Group ID is for mapping of track event to group key $group_id (#5363)
segmentio/segment-docs: src/connections/destinations/catalog/actions-mixpanel/index.md

### Track events are not attributed to Mixpanel Groups

> For Mixpanel (Actions) destination that uses $group_id as the group key, ensure that the mapping(s) handling your `track` events have the field for **Group ID** mapped to a valid value. By default, this field is mapped to the event variable `context.groupId`.

Reviewer: ```suggestion
If the Mixpanel (Actions) destination uses $group_id as the group key, ensure that the mappings handling your `track` events have the field for **Group ID** mapped to a valid value. By default, this field maps to the event variable `context.groupId`.
```
